### ISSUE TITLE
Consider using Yandex' Emoji Translation API to autogenerate bindings

### ISSUE THREAD

**Issue by SaschaMann** [theogf/WatchJuliaBurn.jl#19]
Instead of manually defining bindings, consider using [Yandex Translate's Emoji translation](https://translate.yandex.com/?lang=en-emj&text=throw). They offer an [API](https://translate.yandex.com/developers) that could be used to mass-generate bindings. Obviously without any guarantees of bindings being unique but that adds a fun element of surprise to each function call.

Example:

> WatchJuliaBurn aims at destroying the look of your code by adding emojis like 😄 and kaomojis like c╯°□°ↄ╯ instead of your favorites Julia functions. Don't hesitate to add your worst creations via PR.

=>

> WatchJuliaBurn 🎯  💥  👀  👆  📟  ➕  🐢  💖  😄 & 🐼  💖  🅱  🇨🇬  🌡  🇨🇬  🌡 ↄ 🇨🇬  ⏭  👆  💕  👩 📛  🔢 . 🚫  😥  ➕  👆  😨  🛠  ↗️  🇵🇷.

I expect that this would work just as well for individual bindings.

**Comment by mschauer** [comment]
Careful, your Yandex broke Julia's [aniconism](https://en.wikipedia.org/wiki/Aniconism) tabu. You really want to see the world burn.

```
julia> using ⌚👩🔥
👩> 🍰
π = 3.1415926535897...
```

**Comment by theogf** [comment]
TIL...
That sounds like an interesting proposition, but this also ruins all the fun to find new emoji aliases :D

**Comment by SaschaMann** [comment]
> Careful, your Yandex broke Julia's aniconism tabu. You really want to see the world burn.

Good point, I missed that. That could be manually fixed by hardcording `julia` as `∴` (U+2234) or by asking Yandex to fix this obvious bug.

**Comment by adrhill** [comment]
The Yandex API could be used to generate new file names when calling `emojify`.

**Comment by theogf** [comment]
Just  checked and you need to pay to use the API....

**Comment by l0rem** [comment]
I have no idea what this project is about, but to add insult to injury Yandex does not allow using emj-language with their Translations API, so it is only accessible via web-ui
